tests/src/Traits/ApiTestTrait.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEuropa\Tests\CdtClient\Traits;
+
+use GuzzleHttp\Client as HttpClient;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\Middleware;
+use GuzzleHttp\Psr7\HttpFactory;
+use OpenEuropa\CdtClient\ApiClient;
+use OpenEuropa\CdtClient\Contract\ApiClientInterface;
+use OpenEuropa\CdtClient\Contract\RestInterface;
+use OpenEuropa\CdtClient\Http\Rest;
+use OpenEuropa\CdtClient\Model\Response\Token;
+
+/**
+ * Trait ApiTestTrait
+ *
+ * Provides helper methods for testing classes that utilize ApiClient and Rest.
+ */
+trait ApiTestTrait
```

Comment:
Some of this code was part of the `ClientTestTrait` which has been removed. This trait helps us keep the endpoint tests clean.